### PR TITLE
@New Feature: add shortcut "q" for deft-mode to quit window

### PR DESCRIPTION
**Pull request by TinySong**

### CODE REVIEW

File: layers/+tools/deft/README.org [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-- [[#install][Install]]
-  - [[#layer][Layer]]
-  - [[#configuration][Configuration]]
-- [[#key-bindings][Key Bindings]]
+ - [[#description][Description]]
+   - [[#differences-from-default][Differences from default]]
+ - [[#install][Install]]
+   - [[#layer][Layer]]
+   - [[#configuration][Configuration]]
+ - [[#key-bindings][Key Bindings]]
```
Comment (bmag) [inline layers/+tools/deft/README.org]: What's up with the extra spaces? Please remove them.

**Comment by TinySong** [comment]
PR again, I don't why spacemacs will auto indent extra spaces, do you know why?@bmag, I realy preciate it

**Comment by bmag** [comment]
I don't know what caused the extra spaces, it depends on your configuration and what keys you pressed. I'll merge this later today/tomorrow.

**Comment by bmag** [comment]
Thank you :+1: Cherry-picked into develop, you can safely delete your branch. I squashed the two commits into a single commit and removed the `@` from the commit message.

**Pull request closed without merging**